pipeline/05-finalize.R

```diff
@@ -398,13 +416,19 @@ if (params$toggle$upload_to_s3) {
     paths$output$timing$local,
     paths$output$timing$s3
   )
+
+  # Upload performance report
+  aws.s3::put_object(
+    paths$output$report$local,
+    paths$output$report$s3
+  )
```

Comment:
Sample report [here](https://s3.console.aws.amazon.com/s3/object/ccao-model-results-us-east-1?region=us-east-1&prefix=report/year%3D2023/2023-11-28-dazzling-dan.html). Accessing the report itself is a bit annoying; since all objects in `ccao-model-results-us-east-1` must be private, we can't simply click the `Object URL` link to download the report, and instead we have to run `aws s3 cp` to get a copy. If we're willing to accept the slight increase in security risk, it might make sense to add an access policy specifying that any `*.html` object under the `report/` prefix should be public.

Reply:
**suggestion:** Could we generate a pre-signed S3 URL to include with the output with a time expiration? If that's easy to do then it seems like the perfect fit for this use case.

Reply:
This is a great idea! Unfortunately I couldn't get it to work in ~2 hours of work so I abandoned the idea. It works fine via the `aws s3 presign` CLI, but something about `paws.storage::s3()$generate_presigned_url()` doesn't work properly in either RStudio or the ECS task container environment and was throwing me various opaque access denied errors in spite of everything meeting spec. I think we should punt on this unless/until the current workflow becomes bothersome; hopefully the extra step to navigate to the console page and click the Download button won't be too bad.